account_manager.py

```diff
@@ -2,14 +2,18 @@
 
 class AccountManager:
     def __init__(self):
-        self.connection = psycopg2.connect(
-            database="bank_system",
-            user="postgres",
-            password="Yash@2000",  # Your PostgreSQL password
-            host="localhost",
-            port="5432"
-        )
-        self.cursor = self.connection.cursor()
+        try:
+            self.connection = psycopg2.connect(
+                database="bank_system",
+                user="postgres",
```

Comment:
create separate config file for credentials from there import details
every user will setup db on her/his machine and update the config file to run this configuration